sympy/printing/pretty/pretty.py

```diff
@@ -2273,35 +2273,42 @@ def pretty(expr, **settings):
         pretty_use_unicode(uflag)
 
 
-def pretty_print(expr, **settings):
+def pretty_print(expr, wrap_line=True, num_columns=None, use_unicode=None,
+                 full_prec="auto", order=None, use_unicode_sqrt_char=True):
     """Prints expr in pretty form.
 
     pprint is just a shortcut for this function.
 
-
     Parameters
     ==========
 
     expr : expression
         the expression to print
-    wrap_line : bool, optional
-        line wrapping enabled/disabled, defaults to True
-    num_columns : int or None, optional
+
+    wrap_line : bool, optional (default=True)
+        line wrapping enabled/disabled
+
+    num_columns : int or None, optional (default=None)
         number of columns before line breaking (default to None which reads
         the terminal width), useful when using SymPy without terminal.
-    use_unicode : bool or None, optional
+
+    use_unicode : bool or None, optional (default=None)
         use unicode characters, such as the Greek letter pi instead of
         the string pi.
-    full_prec : bool or string, optional
-        use full precision. Default to "auto"
-    order : bool or string, optional
+
+    full_prec : bool or string, optional (default="auto")
+        use full precision.
+
+    order : bool or string, optional (default=None)
         set to 'none' for long expressions if slow; default is None
-    use_unicode_sqrt_char : bool, optional
+
+    use_unicode_sqrt_char : bool, optional (default=True)
         use compact single-character square root symbol (when unambiguous);
```

Comment:
Looks good, except for ending punctuation: the semicolon ; at the end appears to be hanging. Also, some of the parameter descriptions end with a period . while others do not. Either way is fine with me, but they should be consistent.